README.md

```diff
@@ -126,8 +122,6 @@ This project assumes you have some standard tools to help assemble the project.
  * Hand Drill or Drill Press
  * Dremel, Band saw, or hand saw
  * Items for operating a Raspberry Pi (Keyboard, mouse, monitor, 5V micro USB power adapter)
- 
- Here is [an Amazon list of some of the tools that may be useful](http://a.co/gnoN2LW)
```

Comment:
The amazon list includes wire crimpers. I'm fine with removing the link to the amazon list, but the crimpers ought to be added under optional tools

Reply:
https://www.amazon.com/dp/B07DWVSR41/?coliid=ISMG8XMBILGFK&colid=1IM7RHK25RIBB&psc=0&ref_=lv_vv_lig_dp_it was the item

Reply:
Done - FYI those aren't crimpers, they're strippers. Crimpers are for crimps :p

Reply:
Indeed.